src/components/admin/jokes/JokeItem.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <div class="joke-item">
+      <div class="box">
+        <article class="media">
+          <div class="media-content">
+            <div class="content">
+                <p><small>{{joke.category}}</small>
+                <p>{{joke.content}}</p>
+            </div>
+          </div>
+        </article>
+      </div>
+    </div>
+</template>
+<script>
+    export default {
+        name: 'joke-item',
+        props: {
+            joke: Object
+        }
+    }
+</script>
+<style scoped>
+    .joke-item {
+        background-color: #fff;
+        color: #373737;
+        box-shadow: 0 2px 3px rgba(10,10,10,.1), 0 0 0 1px rgba(10,10,10,.1);
```

Comment:
Weźmy wartości kolorów i paddingu z style.css. Jeżeli tam czegoś brakuje, to dodać